src/content/configuration/other-options.md

```diff
@@ -60,29 +61,174 @@ This will force webpack to exit its bundling process.
+  }
 };
 ```
 
-W> Don't share the cache between calls with different options.
+W> The final location of the cache is a combination of `cache.cacheDirectory` + `cache.name`.
 
-?> Elaborate on the warning and example - calls with different configuration options?
+### `cache.hashAlgorithm`
+
+`string`
+
+Algorithm used the hash generation. See [Node.js crypto](https://nodejs.org/api/crypto.html) for more details. Defaults to `md4`.
+
+`cache.hashAlgorithm` option is only available when [`cache.type`](#cache-type) is set to `filesystem`.
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  cache: {
+    type: 'filesystem',
+    hashAlgorithm: 'md4'
+  }
+};
+```
+
+### `cache.loglevel`
+
+`string: 'debug' | 'info' | 'verbose' | 'warning'`
+
+`cache.loglevel` tells webpack how much of `cache` log info to display.
+
+- `'debug'`: all access and errors with stack trace
+- `'info'`: all access
+- `'verbose'`: all write access
+- `'warning'`: only failed serialization and deserialization
+
+`cache.loglevel` option is only available when [`cache.type`](#cache-type) is set to `filesystem`.
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  cache: {
+    type: 'filesystem',
+    loglevel: 'debug'
+  }
+};
+```
+
+T> Caching usually doesn't emit errors. All failures are warnings and fall back to not caching this item. The build will keep working. It makes sense to enable warnings if you want to investigate why caching isn't working or doesn't increase performance. It also makes sense when developing webpack plugins that affect caching. When reporting bugs for the filesystem cache, make sure to get the stack trace of warnings with the `loglevel: 'debug'` option.
+
+### `cache.name`
+
+`string`
+
+Name for the cache. Different names will lead to different coexisting caches. Defaults to `${config.name}-${config.mode}`. Using `cache.name` makes sense when you have multiple configurations which should have independent caches.
+
+`cache.name` option is only available when [`cache.type`](#cache-type) is set to `filesystem`.
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  cache: {
+    type: 'filesystem',
+    name: 'AppBuildCache'
+  }
+};
+```
+
+### `cache.store`
+
+`string: 'background' | 'idle' | 'instant' | 'pack'`
+
+`cache.store` tells webpack when to store data on the file system. Defaults to `'idle'`.
+
+- `'background'`: Store data in background while compiling, but doen't block the compilation
+- `'idle'`: Store data when compiler is idle in one file per cached item
+- `'instant'`: Store data when instantly. Blocks compilation until data is stored
+- `'pack'`: Store data when compiler is idle in a single file for all cached items
+
+`cache.store` option is only available when [`cache.type`](#cache-type) is set to `filesystem`.
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  cache: {
+    type: 'filesystem',
+    store: 'background'
```

Comment:
```suggestion
    store: 'pack'
```

background and instant make rarely sense, but pack is useful and boosts performance in many cases.